controllers/etcdcluster_controller_test.go

```diff
@@ -140,10 +163,43 @@ func fakeEtcdForEtcdCluster(etcdCluster etcdv1alpha1.EtcdCluster) *StaticRespons
 	return &StaticResponseEtcdAPI{Members: members}
 }
 
+// triggerReconcile forces a Reconcile by making a trivial change to the annotations
+// of the supplied object.
+// A work around for https://github.com/improbable-eng/etcd-cluster-operator/issues/76
+func (s *controllerSuite) triggerReconcile(obj runtime.Object) error {
+	m := meta.NewAccessor()
+	updated := obj.DeepCopyObject()
+	annotations, err := m.Annotations(updated)
+	if err != nil {
+		return err
+	}
+	if annotations == nil {
+		annotations = map[string]string{}
+	}
+	value, found := annotations["last-trigger-value"]
+	if !found {
+		value = "0"
+	}
+	index, err := strconv.Atoi(value)
+	if err != nil {
+		return err
+	}
+	value = strconv.Itoa(index + 1)
+	annotations["last-trigger-value"] = value
+	annotations["last-trigger-key-"+value] = value
```

Comment:
This extra annotation, but I wasn't convinced that simply changing the annotation value was triggering a metadata.resourceVersion change, which is needed to trigger the Reconcile function.

Reply:
Yeah, it wasn't necessary. I wasn't calling triggeringReconcile after errors. Moved it to a defer statement.